research-hub-web/src/app/components/fundings/fundings.component.html

```diff
@@ -1,192 +1,257 @@
+                  >
+                  <span
+                    *ngIf="item.size * 0.001 < 1000 && item.size * 0.001 >= 1"
+                    class="file-size"
+                    >{{ item.size * 0.001 | number: "1.0-0" }}&nbsp;kB</span
+                  >
+                  <span *ngIf="item.size * 0.001 >= 1000" class="file-size"
+                    >{{ item.size * 0.000001 | number: "1.0-0" }}&nbsp;MB</span
+                  >
+                </div>
+              </mat-card-content>
+            </mat-card>
+          </a>
+        </div>
+      </div>
+    </div>
+    <ng-template #noApplicationDocs>
+      <p>It looks like there aren't any documents here.</p>
     </ng-template>
 
-    <!-- Banner -->
-    <ng-container *ngTemplateOutlet="dynamicBanner"></ng-container>
+    <!-- Related -->
+    <div class="funding-owners section" fxLayout="column">
+      <h3>
+        Explore Related
+        <hr />
+      </h3>
+      <div fxLayout="row wrap" fxLayout.xs="column" fxLayoutGap="25px">
+        <app-standard-card
+          id="you-might-be-interested-in"
+          *ngFor="let content of funding.relatedItemsCollection.items"
+          [contentItem]="content"
+        ></app-standard-card>
+      </div>
 
-    <div class="site-padding margin-bottom" ngClass.lt-md="site-padding-mobile top-padding">
+      <h3 *ngIf="funding?.relatedContactsCollection.items.length > 0">
+        Contacts
+        <hr />
+      </h3>
+      <div fxLayout="row wrap" fxLayout.xs="column" fxLayoutGap="25px">
+        <app-contact-card
+          *ngFor="let person of funding.relatedContactsCollection.items"
+          [person]="person"
+        ></app-contact-card>
+      </div>
 
-        <!-- Breadcrumbs -->
-        <ng-container *ngTemplateOutlet="breadcrumbs"></ng-container>
+      <h3 *ngIf="funding?.relatedOrgsCollection.items.length > 0">
+        Organisations
+        <hr />
+      </h3>
+      <div fxLayout="row wrap" fxLayout.xs="column" fxLayoutGap="25px">
+        <app-org-unit-card
+          *ngFor="let orgUnit of funding.relatedOrgsCollection.items"
+          [orgUnit]="orgUnit"
+        ></app-org-unit-card>
+      </div>
 
-        <div>
-
-            <!-- Title and Summary -->
-            <div fxLayout="column wrap">
-                <div><ng-container *ngTemplateOutlet="titleSummary"></ng-container></div>
-
-                <!-- Call To Action -->
-                <div *ngIf="funding.callToAction">
-                    <a href="{{ funding.callToAction }}" target="_blank" mat-flat-button class="standard-button section">
-                        <span *ngIf="funding.callToActionLabel">{{ funding.callToActionLabel }}</span>
-                        <span *ngIf="!funding.callToActionLabel">View Funding</span>
-                    </a>
-                </div>
-                <div class="mini-hr" *ngIf="!funding.callToAction"><hr/></div>
-            </div>
-
-            <!-- Funding Purpose -->
```

Comment:
Missed this section! - Funding purpose is another rich text field at the start of funding pages